app/views/campaigns/peer_to_peer.html.erb

```diff
@@ -9,7 +9,7 @@
 <% end %>
 
 <%= content_for :recaptcha_js do %>
-	<script src="https://www.recaptcha.net/recaptcha/api.js?render=<%= ENV['RECAPTCHA_SITE_KEY'] %> "></script>
+	<script src="https://www.recaptcha.net/recaptcha/enterprise.js?render=<%= ENV['RECAPTCHA_SITE_KEY'] %> "></script>
```

Comment:
According to Google, we need to replace `api.js` with `enterprise.js`. We probably could get this generated by the recaptcha gem so maybe I'll do that.

Reply:
I actually verified that no helper for this exists so I'll resolve this.